poolqc/pooling_qc_sample.py

```diff
@@ -190,7 +191,7 @@ def Summary_table(run_alias, lane_alias):
     downloadtimedate = datetime.today().strftime('%Y-%m-%d')
     download = 'PoolQC_%s_%s_%s.csv' % (downloadtimedate, run_alias, lane_alias)
 
-    return columns, data, style_data_conditional, csv, download
+    return columns, data, style_data_conditional, 2, csv, download
```

Comment:
Is there any way to specify this in the table's properties at creation, instead of continually updating it dynamically with the same value?